{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -91,16 +92,34 @@ def test_approve_transfer(nft, owner, receiver):
     assert nft.balanceOf(receiver) == 0
     assert nft.balanceOf(owner) == 1
     assert nft.ownerOf(1) == owner.address
+    
     with ape.reverts():
         nft.approve(receiver, 1, sender=receiver)
         nft.transferFrom(owner, receiver, 1, sender=receiver)
     assert nft.balanceOf(receiver) == 0
     assert nft.balanceOf(owner) == 1
     assert nft.ownerOf(1) == owner.address
+
     nft.approve(receiver, 1, sender=owner)
     assert nft.getApproved(1) == receiver
     nft.transferFrom(owner, receiver, 1, sender=receiver)
     assert nft.balanceOf(receiver) == 1
     assert nft.balanceOf(owner) == 0
     assert nft.ownerOf(1) == receiver.address
 
+
+def test_uri(nft, owner):
+    {%- if cookiecutter.updatable_uri == 'y' %}
+    assert nft.baseURI() == "dummy uri"
+    nft.mint(owner, sender=owner)
+    
+    
+    nft.setBaseURI("new base uri", sender=owner)
+    assert nft.baseURI() == "new base uri"
+    assert nft.tokenURI(1) == "new base uri/1"
+    
+    {%- else%}
+    assert nft.baseURI() == "dummy uri"
+    nft.mint(owner, sender=owner)
+    assert nft.tokenURI(1) == nft.baseURI() + "/1"
```

Comment:
And change this one:
```suggestion
    assert nft.tokenURI(1) == "dummy uri/1"
```